packages/samples/headless-react/src/components/category-facet/category-facet.class.tsx

```diff
@@ -1,34 +1,35 @@
-import {Component} from 'react';
```

Comment:
The changes in every `.class` and `.fn` files are the same, I'll leave a comment at every significant difference, you can assume every area that doesn't have a comment is a repetition.